### ISSUE TITLE
Show Fancybox preview on asset element index aswell

### ISSUE THREAD

**Issue by tombauer** [boboldehampsink/assetpreview#3]
I just rewrited the code of your assetpreview.js file to get work the plugin also in Asset element index. It also supports later selected assets now. Here is my modified script:

```
    (function($) {
        $(function() {
            // Keep state
            var state = 0;

            $("body").delegate("*", "keydown", function(e) {
                // Listen for space 
                if(e.which == 32) {
                    e.stopPropagation();
                    var myThis;

                    // select the element
                    myThis = $(this).find(".element.hasthumb");
                    if (!myThis.hasClass("hasthumb")){
                        myThis = $(this).closest(".element.hasthumb");
                    }
                    // Toggle
                    if(state) {
                        // Close all boxes
                        $.fancybox.close();
                        state = 0;
                    }else {
                        // Open fancybox
                        $.fancybox($(myThis), {
                            href: $(myThis).data('url'),
                            title: $(myThis).data('label'),
                            keys: {close: [32]},
                            afterShow:  function(){
                                $(".fancybox-close").click(function(){
                                    state = 0;
                                });
                            }   
                        });

                        state = 1;
                    }
                }
            });
        });
    })(jQuery);
```

Unfortunately, the gallery browsing (added in 0.2) does not longer work with my modifications. What do you think? Do you see a better solution adding also the gallery browsing?

**Comment by boboldehampsink** [comment]
Hi, you're using deprecated jQuery functions like `delegate`. Please rewrite and submit a Pull Request if you have any code that you want to get in.

**Comment by tombauer** [comment]
Thanks for the info. I updated the deprecated jQuery function, added and optimized the gallery function and created a Pull Request.

**Comment by boboldehampsink** [comment]
Just released 0.3.0
